task/upload.go

```diff
@@ -626,3 +646,192 @@ func catalystTaskAttemptID(task *api.Task) string {
 	// something more sophisticated in the future.
 	return strconv.Itoa(task.Status.Retries)
 }
+
+func getFile(ctx context.Context, osSess drivers.OSSession, cfg ImportTaskConfig, params api.UploadTaskParams, vodDecryptPrivateKey string) (name string, size uint64, content io.ReadCloser, err error) {
```

Comment:
I kinda liked having these lower level helpers separate not to clutter the main task logic (which is already huge). WDYT of keeping these in a helper file like `files.go` or sth like that?

(btw other than this comment, I'm just assuming the content is exactly the same as transcode.go funcs)